### PR TITLE
Add changelog entries for pending htop-3.2.0 release, update version

### PR DESCRIPTION
**Pull request by natoscott**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,37 @@
+What's new in version 3.2.0
+
+* Support for displaying multiple tabs in the user interface
+* Set correct default sorting direction (defaultSortDesc)
+* Improvements to process list tree sorting
```
Comment (BenBE) [inline ChangeLog:5]: Which PR was this? I don't think this relates to #939.

Comment (natoscott) [reply ChangeLog:5]: Ah, this was in reference to the series of commits from Denis Lisov around December 13, 16, 19 and again on Jan 7th.

Comment (BenBE) [reply ChangeLog:6]: Ah, those hashtable fixes …

The current wording may be confusing with the still pending sorting/removal performance fixes.

Maybe reword?
Improve performance for process lookup and update

Comment (natoscott) [reply ChangeLog]: Done, thanks @BenBE

**Comment by natoscott** [comment]
> Missing:
> 
>     * Allow multiple fixed strings for filters and search (logical OR, separate by "|")
> 

Updated, thanks @fasterit
